config/routes.rb

```diff
@@ -12,5 +12,5 @@
     end
   end
 
-  resources :user_targets, only: :index
+  resources :user_targets, only: [:create, :index]
```

Comment:
Please, stick to our Ruby style guides https://github.com/rootstrap/guides/tree/master/ruby#percent-i